pkg/util/rand/rand.go

```diff
@@ -73,13 +73,80 @@ func Perm(n int) []int {
 // We omit vowels from the set of available characters to reduce the chances
 // of "bad words" being formed.
 var alphanums = []rune("bcdfghjklmnpqrstvwxz0123456789")
+var vowels = []rune("aeiou")
+var consonants = []rune("bcdfghjkmnpqrstvwxyz")
+var alphabets = []rune("abcdefghijkmnpqrstuvwxyz")
+var numbers = []rune("0123456789")
+
+// Returns a random rune from the given rune list
+func randElem(set []rune) rune {
+	return set[Intn(len(set))]
+}
 
 // String generates a random alphanumeric string, without vowels, which is n
 // characters long.  This will panic if n is less than zero.
 func String(length int) string {
 	b := make([]rune, length)
 	for i := range b {
-		b[i] = alphanums[Intn(len(alphanums))]
+		b[i] = randElem(alphanums)
+	}
+	return string(b)
+}
+
+// Returns a rune list of length 'length' with alternating
+// vowels and consonants. Randomly starting from any of the two.
+// If length is 1, returns any random alphabet
+func getAltRune(length int) []rune {
+	b := make([]rune, length)
+	if length == 1 {
+		b[0] = randElem(alphabets)
```

Comment:
Rather than keeping a special case for this, why not just make the loop below understand 1 as a legitimate value and select it the same way?